rules/tines_rules/tines_story_items_destruction.yml

```diff
@@ -0,0 +1,23 @@
+AnalysisType: rule
+DisplayName: "Tines Story Items Destruction"
+Enabled: true
+Filename: tines_story_items_destruction.py
+Severity: Low
+Tests:
```

Comment:
We need a test case that has `ExpectedResult: false` to demonstrate that this does not alert on every log event passed into it.